demos/interactive_face_detection_demo/cpp/README.md

```diff
@@ -69,42 +69,42 @@ omz_converter --list models.lst
-    -no_smooth                 Optional. Do not smooth person attributes
-    -no_show_emotion_bar       Optional. Do not show emotion bar
-    -u                         Optional. List of monitors to show initially.
+    [ -h]                                         show the help message and exit
+    [--help]                                      print help on all arguments
+      -m <MODEL FILE>                             path to an .xml file with a trained Face Detection model
+    [ -i <INPUT>]                                 an input to process. The input must be a single image, a folder of images, video file or camera id. Default is 0
+    [--bb_enlarge_coef <NUMBER>]                  coefficient to enlarge/reduce the size of the bounding box around the detected face. Default is 1.2
+    [ -d <DEVICE>]                                specify a device to infer on (the list of available devices is shown below). Use '-d HETERO:<comma-separated_devices_list>' format to specify HETERO plugin. Use '-d MULTI:<comma-separated_devices_list>' format to specify MULTI plugin. Default is CPU
+    [--dx_coef <NUMBER>]                          coefficient to shift the bounding box around the detected face along the Ox axis
+    [--dy_coef <NUMBER>]                          coefficient to shift the bounding box around the detected face along the Oy axis
+    [--fps <NUMBER>]                              maximum FPS for playing video
+    [--lim <NUMBER>]                              number of frames to store in output. If 0 is set, all frames are stored. Default is 1000
+    [--loop]                                      enable reading the input in a loop
+    [--mag <MODEL FILE>]                          path to an .xml file with a trained Age/Gender Recognition model
+    [--mam <MODEL FILE>]                          path to an .xml file with a trained Antispoofing Classification model
+    [--mem <MODEL FILE>]                          path to an .xml file with a trained Emotions Recognition model
+    [--mhp <MODEL FILE>]                          path to an .xml file with a trained Head Pose Estimation model
+    [--mlm <MODEL FILE>]                          path to an .xml file with a trained Facial Landmarks Estimation model
+    [ -o <OUTPUT>]                                name of the output file(s) to save
+    [ -r]                                         output inference results as raw values
+    [--show] ([--noshow])                         (don't) show output
+    [--show_emotion_bar] ([--noshow_emotion_bar]) (don't) show emotion bar
+    [--smooth] ([--nosmooth])                     (don't) smooth person attributes
+    [ -t <NUMBER>]                                probability threshold for detections. Default is 0.5
+    [ -u <DEVICE>]                                resource utilization graphs. Default is cdm. c - average CPU load, d - load distribution over cores, m - memory usage, h - hide
```

Comment:
```suggestion
    [ -u <DEVICE>]                                resource utilization graphs. Default is cdm. c - average CPU load, d - load distribution over cores, m - memory usage
```
[Here](https://github.com/openvinotoolkit/open_model_zoo/pull/3172#discussion_r814029106)  @Wovchena  said

> h is a key binding. But it can't be provided with -u

But I still think, that `-u h` would be a better way to hide monitors than ` -u ""`

Reply:
Ok, @ieliz, please add `h` support in addition to ""